Assets/MixedRealityToolkit/_Core/Utilities/UX/HoverLight.cs

```diff
@@ -19,9 +20,13 @@ public class HoverLight : MonoBehaviour
         [SerializeField]
         private Color color = new Color(0.3f, 0.3f, 0.3f, 1.0f);
 
-        private int hoverPositionId;
-        private int hoverRadiusId;
-        private int hoverColorId;
+        // Three hover lights are supported at this time.
+        private const int hoverLightCount = 3;
+        private const int hoverLightDataSize = 2;
+        private static List<HoverLight> activeHoverLights = new List<HoverLight>(hoverLightCount);
+        private static Vector4[] hoverLightData = new Vector4[hoverLightCount * hoverLightDataSize];
+        private static int _HoverLightDataID;
+        private static int lastHoverLightUpdate = -1;
 
         public float Radius
```

Comment:
Change to c# 6 auto property :D (example in the old PR)
```public float Radius => radius;```

Also change ordering of fields, Public and then private
Same changes for Color property
(Note, you'll need to change the use of the public properties inside the script to use the private ones)